src/global.css

```diff
@@ -1,2 +1,12 @@
 @import url("https://fonts.googleapis.com/css2?family=IBM+Plex+Serif:ital,wght@0,700;1,700&display=swap");
 @import "~@zanichelli/albe-design-tokens/dist/tokens.css";
+
+z-button button[slot="element"] z-icon:first-child,
+z-button a[slot="element"] z-icon:first-child {
+  margin-right: var(--space-unit);
+}
+
+z-button button[slot="element"] z-icon:not(:first-child),
+z-button a[slot="element"] z-icon:not(:first-child) {
+  margin-left: var(--space-unit);
+}
```

Comment:
Come mai questo stile abbiamo deciso di metterlo qui e non incapsularlo in qualche modo nel componente?

Reply:
Ho risposto sotto a @ufollettu sulla stessa questione